### ISSUE TITLE
[Android/Java] Ambiguous Object

### ISSUE THREAD

**Issue by tgraupmann** [swagger-api/swagger-codegen#4279]
Object has a namespace collision in JsonUtil.java

```java
  public static String serialize(Object obj){
    return getGson().toJson(obj);
  }
```

##### Description

Need to generate the namespace with Object to avoid collision.

##### Swagger-codegen version

Using: http://editor.swagger.io/#/

##### Swagger declaration file content or url

Loading from URL:
https://polling.arena.razerzone.com/swagger/docs/v1

Exporting to Android.

Importing to Android Studio.

Gradle reports the ambiguous Object error.

##### Steps to reproduce

1. Go to http://editor.swagger.io/

2. File Import URL:
https://polling.arena.razerzone.com/swagger/docs/v1

3. Generate Client Android

4. Unpack zip file

5. Import as new project in Android Studio

6. See the gradle compile error

##### Related issues

https://github.com/swagger-api/swagger-codegen/issues/4278

##### Suggest a Fix

```java
  public static String serialize(java.lang.Object obj){
    return getGson().toJson(obj);
  }
```

```java
  public ResponseOutputModelString authenticateGetRazerIdAccessTokenProfile (RazerIdTokenLoginPostModel tokenLoginModel) throws TimeoutException, ExecutionException, InterruptedException, ApiException {
     java.lang.Object postBody = tokenLoginModel;
```

```java
  @Override
  public boolean equals(java.lang.Object o) {
```

**Comment by tgraupmann** [comment]
I made a Windows.Forms app to automate syntax fixes as a workaround. The ideal fix would be to fix the template. https://github.com/tgraupmann/SwaggerAndroidClientFixer

**Comment by tgraupmann** [comment]
Trying to add Client: Android label...

**Comment by wing328** [comment]
@tgraupmann using namespace is one way to resolve the issue. I've instead added "Object" as a keyword for the time being so the model will be renamed to "ModelObject" instead: #4305

**Comment by wing328** [comment]
Closed via #4305 

Please pull the latest master to give it a try.
